src/api/package-manager-api.ts

```diff
@@ -0,0 +1,76 @@
+import {httpClientV2} from "../services/http-client-service.v2";
+
+export interface AssetMetadataTransport {
+    hidden: boolean;
+}
+
+export interface ContentNodeTransport {
+    id: string;
+    key: string;
+    name: string;
+    rootNodeKey: string;
+    assetType: string;
+    nodeType: string;
+    parentNodeId: string;
+    invalidContent: string;
+    serializedContent: string;
+    serializationType: string;
+    draftId: string; // TODO - Delete after the external services have adapted to the change
```

Comment:
Let's keep here only the fields we are explicitly using in the CLI. Also the comment should not be needed.